zygrader/config/versioning.py

```diff
@@ -107,6 +107,14 @@ def versioning_update_preferences():
     if compare_versions(version, user_version):
         config["spooky_mode"] = False
 
+    version = "5.0.0"
+    if compare_versions(version, user_version):
+        config["unicode_mode"] = False
+
+    version = "5.0.1"
+    if compare_versions(version, user_version):
+        config["theme"] = "Default"
+
```

Comment:
You can delete this.

Editing "old" versioning code doesn't work. For example, most (hopefully all) TAs have used version 5.0.0 at this point. So this if statement wouldn't run for any of those students because they already have updated to that version.

Additionally, I wrote `update_config_with_new_preferences` which does this automatically for new preference variables.

Reply:
Should I delete the versioning for 5.0.0 as well? All of it, goes then?

Reply:
It's safe to remove all of it. `update_config_with_new_preferences` will always add new defaults